packages/host/app/components/matrix/forgot-password.gts

```diff
@@ -61,8 +68,11 @@ export default class ForgotPassword extends Component<Signature> {
           {{on 'click' this.sendEmailValidation}}
         >Reset Your Password</Button>
         <span class='or'>or</span>
-        <Button class='button' data-test-cancel-btn {{on 'click' @onLogin}}>Back
-          to login</Button>
+        <Button
+          class='button'
+          data-test-cancel-btn
```

Comment:
This test selector could be named in a way that is more specific to this component